### ISSUE TITLE
Add missing hints for `org.thymeleaf.expression.Temporals`

### ISSUE THREAD

**Issue by sdeleuze** [oracle/graalvm-reachability-metadata#95]
I will work on this.

**Comment by sdeleuze** [comment]
Fixed by #96.

**Comment by czp3009** [comment]
I apologize for leaving a comment on a long ago issue, but I'm really confused

This issue add hints for `org.thymeleaf.expression.Temporals`, but there are still a few express objects in the same package that have not been added:

```
org.thymeleaf.expression.Conversions
org.thymeleaf.expression.Lists
```

Is this done on purpose or simply missed?
